src/Organisms/NavElement/NavElement.js

```diff
@@ -14,15 +14,21 @@ import { useState } from 'react';
 const NavElement = () => {
   const locationPath = useLocation();
   const foundPath = pathData.find((e) => e.path === locationPath.pathname);
-
+  const login = locationPath.pathname === '/login';
+  const register = locationPath.pathname === '/register';
+  const contact = locationPath.pathname === '/contact-form';
   const [open, setOpen] = useState(false);
 
   return (
     <>
       <NavElementStyled>
-        <div className="burger-wrapper">
-          <MdMenu className="burger-icon" onClick={() => setOpen(true)} />
-        </div>
+        {login || register || contact ? (
+          <div className="burger-wrapper" />
+        ) : (
+          <div className="burger-wrapper">
+            <MdMenu className="burger-icon" onClick={() => setOpen(true)} />
+          </div>
+        )}
```

Comment:
it works nice!
add it also to /forgot and /in-progress

Reply:
If yes- done ;)

Reply:
exactly that, great!